docs/schema/aws.md

```diff
@@ -2158,3 +2174,80 @@ Representation of an AWS [API Gateway Resource](https://docs.aws.amazon.com/apig
         ```
         (APIGatewayRestAPI)-[RESOURCE]->(APIGatewayResource)
         ```
+
+## AutoScalingGroup
+
+Representation of an AWS [Auto Scaling Group Resource](https://docs.aws.amazon.com/autoscaling/ec2/userguide/AutoScalingGroup.html).
+
+| Field | Description |
+|-------|-------------|
+| firstseen| Timestamp of when a sync job first discovered this node  |
+| lastupdated |  Timestamp of the last time the node was updated |
+| **arn** | The ARN of the Auto Scaling Group|
+| name |  The name of the Auto Scaling group. |
+| createdtime | The date and time the group was created. |
+| launchconfigurationname | The name of the associated launch configuration. |
+| maxsize | The maximum size of the group.|
+| minsize | The minimum size of the group.|
+| defaultcooldown | The duration of the default cooldown period, in seconds. |
+| desiredcapacity | The desired size of the group. |
+| healthchecktype | The service to use for the health checks. |
+| healthcheckgraceperiod | The amount of time, in seconds, that Amazon EC2 Auto Scaling waits before checking the health status of an EC2 instance that has come into service.|
+| status | The current state of the group when the DeleteAutoScalingGroup operation is in progress. |
+| newinstancesprotectedfromscalein | Indicates whether newly launched instances are protected from termination by Amazon EC2 Auto Scaling when scaling in.|
+| maxinstancelifetime | The maximum amount of time, in seconds, that an instance can be in service. |
+| capacityrebalance | Indicates whether Capacity Rebalancing is enabled. |
+| region | The region of the auto scaling group. |
+
+### Relationships
+
+- AWS Auto Scaling Groups are a resource under the AWS Account.
+
+        ```
+        (AWSAccount)-[RESOURCE]->(AutoScalingGroup)
+        ```
+- AWS Auto Scaling Groups has one or more subnets/vpc identifiers.
+
+        ```
+        (AutoScalingGroup)-[VPC_IDENTIFIER]->(EC2Subnet)
+        ```
+- AWS EC2 Instances are members of one or more AWS Auto Scaling Groups.
+
+        ```
+        (EC2Instance)-[MEMBER_AUTO_SCALE_GROUP]->(AutoScalingGroup)
+        ```
+
+## EC2Image
+
+Representation of an AWS [EC2 Images (AMIs)](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html).
```

Comment:
Let's also link to the API doc: https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeImages.html

Reply:
Done